columnq/src/columnq.rs

```diff
@@ -40,6 +41,15 @@ impl ObjectStoreProvider for ColumnQObjectStoreProvider {
                     Err(err) => Err(DataFusionError::External(Box::new(err))),
                 }
             },
+            "gs" => {
```

Comment:
this hardcoded value is duplicated with https://github.com/roapi/roapi/blob/b7b79cb1fcb23d686b76aa4dfe68a7be9c67428e/columnq/src/io/mod.rs#L31

I think it is fine for just s3/gs, but not ideal.
Azure have more schemas to support, so I did refactor it to use `io::BlobStorageType` later in Azure draft: https://github.com/jychen7/roapi/pull/5/commits/b65c246659d9c9e054f6f5d3ffa199016acfbb87

Reply:
nice, i was going to suggest implementing a from trait for BlobStrorageType :D